test/unit/edgeware/testUtil.ts

```diff
@@ -131,6 +131,9 @@ export function constructFakeApi(
       }
     },
     derive: {
+      staking: {
+        validators: callOverrides['validators'],
```

Comment:
If you look above, there's already a callOverride for "validators" -- you'll need to make this something like "validatorsDerive" to avoid object collision.

Reply:
Yeah, missed it, thanks for pointing out.